tests/basic/node_decommission_test.go

```diff
@@ -79,16 +81,49 @@ var _ = Describe("{DecommissionNode}", func() {
 					}
 					decommissioned, err := task.DoRetryWithTimeout(t, defaultTimeout, defaultRetryInterval)
 					log.FailOnError(err, "Failed to get decommissioned node status")
-					dash.VerifyFatal(decommissioned.(bool), true, "Validate node is decommissioned")
+					dash.VerifyFatal(decommissioned.(bool), true, fmt.Sprintf("Validate node [%s] is decommissioned", nodeToDecommission.Name))
 				})
 			})
 			stepLog = fmt.Sprintf("Rejoin node %s", nodeToDecommission.Name)
 			Step(stepLog, func() {
 				log.InfoD(stepLog)
 				err := Inst().V.RejoinNode(&nodeToDecommission)
 				dash.VerifyFatal(err, nil, "Validate node rejoin init")
+				var rejoinedNode *api.StorageNode
+				t := func() (interface{}, bool, error) {
+					drvNodes, err := Inst().V.GetDriverNodes()
+					if err != nil {
+						return false, true, err
+					}
+
+					for _, n := range drvNodes {
+						if n.Hostname == nodeToDecommission.Hostname {
+							rejoinedNode = n
+							return true, false, nil
+						}
+					}
+
+					return false, true, fmt.Errorf("node %s not joined yet", nodeToDecommission.Name)
+				}
+				_, err = task.DoRetryWithTimeout(t, appReadinessTimeout, defaultRetryInterval)
```

Comment:
Are you sure you want to ignore the bool value retuned by the anon function `t`, because you are returning bool there but ignoring it here.

Reply:
ya that is fine